### ISSUE TITLE
11y_VscodeWebviewUiToolkit_ProgressRing_Usability: There is not textual representation of progress ring that how much it is completed

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/vscode-webview-ui-toolkit#211]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yUsable;#A11yTCS;#A11ySev2;#SH_VscodeWebviewUiToolkit_Aug2021;#Webview UI Toolkit for Visual Studio Code;#Win32;#FTP;#DesktopApp;

### Environment Details:
Application Name: Vscode Webview Ui Toolkit
Windows Version: Win10

### Repro Steps:

1. Launch VS Code
2. Open WebView UI Toolkit: All Component extension
3. Tab to "Progress ring"
4. Check if there is textual representation of progress ring that how much it is completed

### Actual:
There is not textual representation of progress ring that how much it is completed. Like for 75% progress, there should be textual representation

### Similar issue is observed with scenarios:

1. Progress bar

### Expected:
There should be textual representation of progress ring that how much it is completed. Like for 75% progress, there should be textual representation

### User Impact:
Cognitive disable users will face difficulty in getting the data
